WordPress/gutenberg
# File block: Add spacing support

## What?

Adds margin and padding block support to the file block.

Moves the default margin from style.scss to block.json to reduce duplicate styles:
- Before this PR: When the default margin for the block is changed in theme.json, both the margin from theme.json and style.scss is output. The style in theme.json overrides the CSS from the style.scss file.
- With this PR: The margin in theme.json replaces the margin from block.json and the duplicate style is not output.

## Why?

For consistency.

## How?


## Testing Instructions
Create a new post or page.
Add two file blocks: one should be an embedded PDF file, the second any linked file.
Confirm that there is a dimension panel and that margin and padding are available but not enabled by default.
Confirm that the default bottom margin is present.
Test the margin and padding settings in the editor and front.

Note:  The File block is not listed in Site Editor > Styles > Blocks but can be styled via theme.json.


## Screenshots or screencast
